recipes/repo_clas_design_template.md

```diff
@@ -0,0 +1,324 @@
+# Users and Peeps Model and Repository Classes Design Recipe
```

Comment:
Very well planned and neat! nice :)

Reply:
Strong agree.

Reply:
I like it you've got both classes in here.